api/meetups/tests/test_update_meetup.py

```diff
@@ -0,0 +1,109 @@
+from rest_framework.reverse import reverse
+from rest_framework import status
+from rest_framework.test import APIClient, APITestCase
+from rest_framework.authtoken.models import Token
+from users.models import User
+from meetups.models import Meetup
+import pytz
+from datetime import datetime
+
+
+class BaseTest(APITestCase):
+    """
+    The base test class for all default test case settings
+    """
+
+    def setUp(self):
+        self.client = APIClient()
+        self.user = User.objects._create_user(
+            name="Issa Mwangi",
+            email="issaianmaina@gmail.com",
+            password="@12345gh"
+        )
+        self.user.is_active = True
+        self.user.save()
+        self.admin = User.objects._create_user(
+            name="Issa Mwangi",
+            email="issamwangi@gmail.com",
+            password="@Admin123"
+        )
+        self.admin.is_active = True
+        self.admin.is_staff = True
+        self.admin.save()
+
+        Meetup.objects.create(
+            title='Test Driven Development',
+            body='Developers need to discusss the correct approach of doing test driven development',
+            location='Andela Campus',
+            creator=self.admin,
+            scheduled_date=pytz.utc.localize(datetime.strptime(
```

Comment:
Hello @IssaIan ,
Instead of using pytz to localize time, django.utils.timezone can help you get a localized timezone. Putting a specific date in the test would mean the test will fail at some point.
Something like `scheduled_date=timezone.now() + timezone.timedelta(1) will help you set scheduled date to the next day, so the test will never fail even in 100 years.

Reply:
Let me fix this